CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### Features
 * Your contribution here.
+* [#2020](https://github.com/ruby-grape/grape/pull/2020): Reduce array allocation - [@ericproulx](https://github.com/ericproulx).
```

Comment:
For next time, you can do `[#2020](https://github.com/ruby-grape/grape/pull/2020), [#2014](https://github.com/ruby-grape/grape/pull/2014): Reduce ...`.